src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -8,7 +8,7 @@
 // associated with an assembly.
 [assembly: AssemblyCompany("Autodesk, Inc")]
 [assembly: AssemblyProduct("Dynamo")]
-[assembly: AssemblyCopyright("Copyright � Autodesk, Inc 2024")]
+[assembly: AssemblyCopyright("Copyright © Autodesk, Inc 2024")]
```

Comment:
This file shouldn't be committed in the PR's

Reply:
There is a change to this file still.

Reply:
I used the exact same line from Github but it is showing as a change, I think this is fine for now